src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -48,6 +50,10 @@ using std::getenv;
 using std::set;
 using std::string;
 using std::vector;
+using std::ifstream;
+using std::istreambuf_iterator;
+
+const char* FILE_ARG_PREFIX = "file:";
```

Comment:
Minor quip, in google c++ style guide consts are called kMyConstant rather than MY_CONSTANT. (not sure if we're following google style guide, but we might as well follow *some* style guide).

Reply:
Thanks - good to learn! Done.